### PR TITLE
fix some bugs

### PR DESCRIPTION
**Pull request by knightXun**
fix some bugs

### CODE REVIEW

File: contributors/design-proposals/instrumentation/events-redesign.md [hunk 1]
```diff
@@ -368,11 +368,11 @@ Another considered solution was to store timestamps of Events explicitly instead
...
 
 ### Using new API group for storing data
-Instead of adding "new" fields to the "old" versioned type, we could have change the version in which we store Events to the new group and use annotations to store "deprecated" fields. This would allow us to avoid having "hybrid" type, as `v1.Events` became, but the change would have a much higher risk (we would have been moving battle-tested and simple `v1.Event` store to new `events.Event` store with some of the data present only in annotations). Additionally performance would degrade, as we'd need to parse JSONs from annotations to get values for "old" fields.
+Instead of adding "new" fields to the "old" versioned type, we could have change the version in which we store Events to the new group and use annotations to store "deprecated" fields. This would allow us to avoid a "hybrid" type, as `v1.Events` became, but the change would have a much high risk (we would have been moving battle-tested and simple `v1.Event` store to new `events.Event` store with some of the data present only in annotations). Additionally performance would degrade, as we'd need to parse JSONs from annotations to get values for "old" fields.
```
Comment (yastij) [inline contributors/design-proposals/instrumentation/events-redesign.md]: Why this is changing. Can you revert this change ?

File: contributors/design-proposals/instrumentation/events-redesign.md [hunk 2]
```diff
@@ -368,11 +368,11 @@ Another considered solution was to store timestamps of Events explicitly instead
...
 Adding panic button that would stop creation/update of Events
 If all other prevention mechanism fail we’d like a way for cluster admin to disable Events in the cluster, to stop them overloading the server. However, we dropped this idea, as it's currently possible to achieve the similar result by changing RBAC rules.
 
-### Pivoting towards more machine readable Events by introducing stricter structure
+### Pivoting towards more machine readable Events by introducing more stricter structure
```
Comment (yastij) [inline contributors/design-proposals/instrumentation/events-redesign.md]: Remove this addition.

Comment (knightXun) [reply contributors/design-proposals/instrumentation/events-redesign.md]: ok
